docs/whats-new/production-release-notes.md

```diff
@@ -2,10 +2,25 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Production release 06 October 2022
+
+Frontend version: 1.0.0-20220927.1052 | Backend version: 1.31.10-220927.0538
+
+**New features** - **Backend**
+
+<details>
+<summary style="font-size:20px;font-weight:bold">We send emails to the requestor and the TechPass account holder when a TechPass account's status changes.</summary>
+
+Won't it be great to know the progress of your request to enable, disable or terminate your or others' TechPass account?
+
+We have introduced a new feature to send an email to the requestor and account holder when a TechPass account is enabled, disabled and terminated. The requestor will be aware of the progress of their request at all stages.
+
+</details>
```

Comment:
this is missing?
<summary style="font-size:20px;font-weight:bold">Fixed incorrect validation rules for public officer's email address.</summary>